src/index.html

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8"/>
-  <base href="/"/>
+  <head>
+    <meta charset="utf-8" />
+    <base href="/" />
 
-  <meta content="Landville" name="author">
-  <meta content="width=device-width, initial-scale=1" name="viewport"/>
-  <title>landville</title>
-  <meta content="" name="description">
-  <!-- Favicon -->
-  <link href="favicon.ico" rel="icon" type="image/x-icon"/>
-  <!-- FontAwesome 4.7.0 -->
-  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
-</head>
-<body>
-<app-root></app-root>
-</body>
+    <meta content="Landville" name="author" />
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <title>landville</title>
+    <meta content="" name="description" />
+    <!-- Favicon -->
+    <link href="favicon.ico" rel="icon" type="image/x-icon" />
+    <!-- FontAwesome 4.7.0 -->
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
```

Comment:
Good work @Genza999, please remember to update the CDN link to use font awesome 5.
It provides more icons and is backward compatible.

Reply:
Thanks @CryceTruly i have worked on that.